### PR TITLE
Popup window 

### PR DESCRIPTION
**Pull request by SamuelTakyiBoateng**
Modified script.js to implement the following
- dynamically load page
- to implement pop-up window for both mobile & desktop version

Refractor and created 
- style-main.css
- style-project.css

Refractor and modified
- index.html

### CODE REVIEW

File: style projects.css [hunk 1]
```diff
@@ -0,0 +1,505 @@
...
+}
+
+/* container for 'Recent projects' title */
+.title {
+  display: grid;
+  padding: 5rem 0;
+  grid-template-columns: 20rem;
+  justify-content: center;
+  text-align: center;
+}
```
Comment (aamirkhan2478) [inline style projects.css:21]: - [ ] Well done :+1:  your implementation is very good :clap: but your design is not the same as Figma design kindly fix your Figma design issue :slightly_smiling_face: check below :point_down: the comparison between your design and Figma design :+1:

**Your Desktop Design**
Screenshot from 2023-04-20 14-54-44

**Figma Desktop Design**
Screenshot from 2023-04-20 14-54-31

**Your Mobile Design**
Screenshot from 2023-04-20 14-51-19

**Figma Mobile Design**
Screenshot from 2023-04-20 14-53-17

- [ ] If you check the Figma desktop version the buttons are at the end of the popup but in your case the buttons are not at the end of the popup and if you check your popup image the size is not the same as Figma design so kindly fix both issues :+1: 
- [ ] If you check your desktop version the buttons are not shown in your design but in the Figma design, there are two buttons at the end of the popup window so kindly fix these issues and kindly stick to the Figma design :slightly_smiling_face:

File: index.html [hunk 2]
```diff
@@ -280,8 +282,8 @@ <h3 class="work2Title">
 								>See project</a
 							>
 						</div>
-					</div>
-				</div>
+					</div> -->
+				<!-- </div> -->
```
[the remark this answers is not in the document]
Comment (SamuelTakyiBoateng) [reply index.html]: Thank you so much for your insightful comments. Please, I have deleted the unwanted code as advised.

File: style projects.css (hunk 1 above)
Comment (SamuelTakyiBoateng) [reply style projects.css:21]: I really appreciate your eye for detail and quest to bring out the very best in me.
Please, I have adjusted the position and listed the buttons below the page for the mobile version. Significant changes have also been made to the desktop version as much as possible. My apologies for the miss 🙏 and thanks again

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle: 

Hi @SamuelTakyiBoateng, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 



 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js [hunk 3]
```diff
@@ -25,3 +25,335 @@ menuItems.forEach((item) => {
...
+/** *********************Modal/Popup Section ********************************* */
+function projectModal(e) {
+  const docFragment = document.createDocumentFragment();
+  const projectCardInfo = modalProjects[parseInt(e.target.dataset.projectbtn, 10)];
+
+  const popupContainer = createElement('div', 'popupContainer');
+
+  const projectPopup = document.createElement('div');
+  projectPopup.classList.add('projectPopup', 'project-card');
+
```
[the remark this answers is not in the document]
Comment (SamuelTakyiBoateng) [reply script.js:294]: I'm sorry for missing this and thankful for identifying it. I have updated it with the right image.
Thanks again🙏

File: style projects.css [hunk 4]
```diff
@@ -0,0 +1,503 @@
...
+/*********************** Modal/Popup Section **********************************/
+.popupContainer {
+  position: fixed;
+  top: 0;
+  left: 0;
+  height: 100vh;
+  width: 100%;
+  z-index: 2;
+}
+
```
Comment (AnselemOdims) [inline style projects.css:336]: - [ ] As pointed out by the previous reviewer, please ensure the modal fills the view width of the page on the mobile version.

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @SamuelTakyiBoateng , 
Good job implementing some of the changes requested by the previous reviewer.

## Required Changes ♻️
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: style projects.css (hunk 4 above)
Comment (SamuelTakyiBoateng) [reply style projects.css:336]: .

Comment (SamuelTakyiBoateng) [reply style projects.css:336]: I am a little confused about making the modal fill the view width of the page on the mobile version. The width for my mobile view seems to be the same as that of the figma mobile view. I have attached those below. Please take a look

Figma mobile view
---------------------------------
figma



My mobile View
-------------------
myscreenshot

**Review by BenMukebo — APPROVED** [review]
### Hi @SamuelTakyiBoateng, 👋 

# STATUS: Approved  🎉 

Great work on making the changes requested by a previous reviewer :1st_place_medal:  
 Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉  🎉 🎉 



### To Highlight :1st_place_medal: 
- Your Pull Request has a proper title and description. ✔️  
- No Linter errors :heavy_check_mark: 
- The projects section is created dynamically with the information from the projects array. 🥇 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._https://raw.githack.com/SamuelTakyiBoateng/Portfolio-RWD-desk_mob/popup-window/index.html

**Pull request merged**